index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    
+    <header class="toolbar">
+     
+       <nav class="logo"><a  href="#"><img src="Images\Logo Placeholder.png" alt="logo" /></a></nav> 
+      <img class="icons" src="Images\Enabled.png" alt="threeline" />
+  </header>
+
+  <section class="sec">
+    <div class="headline">
+    
+        <h1 class="text1">I'm Burak Glad to see you</h1>
+        <p class="text2">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+      
+        <h2><p class="let">LET'S CONNECT</p></h2>
+
+         <ul  class="socialmediaicons" >
+          <li>
+            <a class="social" href="#">
+              <img src="Images\twitter.png" alt="twitter">
+            </a>
+          </li>
+          <li>
+            <a class="social" href="#">
+              <img src="Images\Linkedin icon.png" alt="link"> </a>
+            </a>
+          </li>
+          <li>
+            <a class="social" href="#">
+              <img src="Images\m.png" alt="Angelist"> </a>
+            </a>
+          </li>
+          <li>
```

Comment:
- [ ] You did a great job structuring your social media icons as a list! 🚀 The only thing that's left is for you to add your social media links to each one of the icons inside the `href` attribute! Having these links will provide an easy way for users visiting your page to connect with you. 🤓 🛠️